### ISSUE TITLE
Hibernation is disabled by default

### ISSUE THREAD

**Issue by lourkeur** [nrdxp/nixflk#114]
**Describe the bug**
On a fresh nixos config derived from the core branch, `systemctl hibernate fails w/
```
Failed to hibernate system via logind: Sleep verb "hibernate" not supported
```

**To Reproduce**
1. switch to a configuration
2. check that `security.protectKernelImage` is true
3. reboot
4. attempt to hibernate

**Expected behavior**
Hibernation should be possible just like in vanilla NixOS.

**Additional context**
It took me a bit of research to figure out what the cause was. I would argue that, while there's nothing fundamentally bugged with the option, not being able to hibernate and having no indication as to why isn't good UX. I suggest either
1. yeeting the setting from `profile/core` and possibly move it to a "hardened" profile, ([NixOS has one already](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/profiles/hardened.nix))
2. clearly documenting this fact.

**Comment by nrdxp** [comment]
good catch, I think this is a leftover from the very beginning of the project. Importing the official hardened profile is a better idea. Resolved by eb275d7d0e110beedcd2e52e104d57ccce8db0ec

**Comment by benneti** [comment]
as the profile also contains `hideProcessInformation = true;` and this one interferes with some display managers at the moment, it probably should be disabled, too.
(see https://github.com/NixOS/nixpkgs/issues/111629)
